### PR TITLE
build: fix dockerfile

### PR DESCRIPTION
**Pull request by anonkey**
Fixing docker file build

### CODE REVIEW

File: lib/Cargo.toml
Comment (anonkey) [inline lib/Cargo.toml:45]: There was a dependency error during docker build caused by an AWS related feature thats in defaults in rustls i just removed it

File: Dockerfile
Comment (anonkey) [inline Dockerfile:22]: Necessary to use vendors during build

**Review by FlorentinDUBOIS — APPROVED** [review]
LGTM, waiting for the ci to run before merging :)

**Pull request merged**
